README.md

```diff
@@ -183,6 +183,7 @@ Provisioning.PasswordCryptSaltLength=10
 ResourceDisk.Format=y
 ResourceDisk.Filesystem=ext4
 ResourceDisk.MountPoint=/mnt/resource
+ResourceDisk.MountOptions=None
```

Comment:
@hglkrijger is right. I believe in this case you need to check against str value `"None"`.

@nbirnel why don't you leave the value empty as `ResourceDisk.MountOptions=` then you can simply check it as `if not mountOpts:`